dimensiondata/src/main/java/org/jclouds/dimensiondata/cloudcontrol/domain/Disk.java

```diff
@@ -29,17 +29,19 @@
    @Nullable
    public abstract String id();
 
-   public abstract int scsiId();
+   @Nullable
+   public abstract Integer scsiId();
 
-   public abstract int sizeGb();
+   @Nullable
+   public abstract Integer sizeGb();
```

Comment:
Can this field have decimals? (Just asking; I'm not familiar with the API).

Reply:
In our API disk size is always an integer value.